docs/src/examples/components/Dialog/Content/DialogExampleContent.shorthand.tsx

```diff
@@ -7,6 +7,7 @@ const DialogExampleContent: React.FC = () => (
     confirmButton="Confirm"
     content="Are you sure you want to confirm this action?"
     header="Action confirmation"
+    headerAction={{ icon: 'more', title: 'See more options' }}
```

Comment:
Can we add a separate example instead? 🤔 

And probably `close` icon will have more sense at that place. We can use a knob to handle that:

```tsx
const [open, setOpen] = useBooleanKnob({ name: 'open })

<Dialog
  onOpen={() => setOpen(true)}
  onCancel=...
  onConfirm=...
  headerAction={{ onClick: () => setOpen(true) }}
/>
```

Reply:
Created new example under the Content section.